### PR TITLE
remove intervention policy group form.

### PR DESCRIPTION
**Pull request by Tom-Szendrey**
# Description

- remove generic group form
- utilize static and dynamic group forms directly

### CODE REVIEW

**Review by YohannParis — APPROVED** [review]

File: packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-dynamic-intervention-policy-group.vue [hunk 1]
```diff
@@ -1,20 +1,97 @@
...
+		<div class="form-header">
+			<div>
+				<tera-input v-if="isEditing" v-model="knobs.name" placeholder="Policy bounds" />
+				<h6 v-else>{{ knobs.name }}</h6>
+				<i
+					:class="{ 'pi pi-check i': isEditing, 'pi pi-pencil i': !isEditing }"
+					:style="'cursor: pointer'"
+					@click="onEdit"
+				/>
+			</div>
```
Comment (mwdchang) [inline packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-dynamic-intervention-policy-group.vue:12]: @blanchco created a light-wrapper component that supposedly does this kind of swapping between readonly-text and text-input.

I cannot recall what is the component here, but you may want to see if it is applicable here.

File: packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-static-intervention-policy-group.vue [hunk 2]
```diff
@@ -1,85 +1,103 @@
...
+	<div class="policy-group">
+		<div class="form-header">
+			<div>
+				<tera-input v-if="isEditing" v-model="knobs.name" placeholder="Policy bounds" />
+				<h6 v-else>{{ knobs.name }}</h6>
+				<i
+					:class="{ 'pi pi-check i': isEditing, 'pi pi-pencil i': !isEditing }"
+					:style="'cursor: pointer'"
+					@click="onEdit"
 				/>
```
Comment (mwdchang) [inline packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-static-intervention-policy-group.vue:11]: See https://github.com/DARPA-ASKEM/terarium/pull/4080/files#r1670837039

File: packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-dynamic-intervention-policy-group.vue (hunk 1 above)
Comment (blanchco) [reply packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-dynamic-intervention-policy-group.vue:12]: `tera-toggleable-edit.vue`, an example of its use is the the create interventions operator

Comment (Tom-Szendrey) [reply packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-dynamic-intervention-policy-group.vue]: Good shoutout thank you for this!

File: packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-static-intervention-policy-group.vue (hunk 2 above)
Comment (Tom-Szendrey) [reply packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-static-intervention-policy-group.vue]: Thank you for this!

**Review by blanchco — APPROVED** [review]

**Pull request merged**